README.md

```diff
@@ -75,16 +77,26 @@ logger.error('MISSING_VALUE_ERROR', "Expected 'x' to be present on 'Vector'");
 logger.clearCommonTags();
 ```
 
+#### Log levels
+
+There are four supported levels of logging, which correspond to Python's levels
+of logging:
+
+- `logger.debug` (10)
+- `logger.info` (20)
+- `logger.warning` (30)
+- `logger.error` (40).
```

Comment:
Nit: I don't think this list should be `.` terminated.

Reply:
removed in 5f34456